app/src/main/java/space/taran/arknavigator/utils/ImageUtils.kt

```diff
@@ -65,4 +69,25 @@ object ImageUtils {
             }
         }
     }
+
+    fun <T> glideExceptionListener() = object: RequestListener<T>{
+        override fun onLoadFailed(
+            e: GlideException?,
+            model: Any?,
+            target: Target<T>?,
+            isFirstResource: Boolean
+        ): Boolean {
+            Log.d(GLIDE, "load failed with message: ${e?.message} for target of type: ${target?.javaClass?.canonicalName}")
+            //Glide docs: "true if the listener has handled updating the target for the given exception."
```

Comment:
I think, this commentary is redundant, other developer should read official docs before trying to understand custom code :)